examples/wrapper.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
```

Comment:
I think we're trying to push more things to docs rather than examples, right? Would this example fit under a guide? Can it be demoed in docs easily enough?